cryptfs.go

```diff
@@ -83,15 +95,19 @@ func (fsys *FS) ReadFile(name string) ([]byte, error) {
 
 // Disfigure will encrypt and encode the plaintext
 func (fsys *FS) Disfigure(plaintext []byte) ([]byte, error) {
-	encryptedBytes, err := fsys.cryptor.encrypt(plaintext)
+	bs, err := fsys.compressor.compress(plaintext)
```

Comment:
should we first encrypt it and only then compress it?

Reply:
Compression would save more space if it's the first step. Gzip is designed for written text or document formats where it can use repeated patterns for better compression. Compressing random data does not save as much space.